### PR TITLE
[ Test ] Input Generator @open sesame 06/11 16:53 

### PR DESCRIPTION
**Pull request by jijoongmoon**
This PR provides layer input generator to evalute the layer
calculation.

[ To Do ]
Golden Test Result Generation should be added.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: test/input_gen/genInput.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+def gen_input(input_filename, batch, channel, height, width):
+    if os.path.isfile(input_filename):
+        os.remove(input_filename)
+        
+    for i in range(0,batch):
+        imarray=numpy.random.rand(channel, height, width)*255
+        with open(input_filename, 'ab') as outfile:
+            numpy.array(imarray, dtype=numpy.float32).tofile(outfile)
+        print (i+1, " x ", imarray.shape, " data is generated")
```
Comment (dongju-chae) [inline test/input_gen/genInput.py:24]: Unnecessary space between `print` and `(`?

File: test/input_gen/genInput.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import numpy
+import struct
+
+def gen_input(input_filename, batch, channel, height, width):
+    if os.path.isfile(input_filename):
+        os.remove(input_filename)
+        
+    for i in range(0,batch):
+        imarray=numpy.random.rand(channel, height, width)*255
+        with open(input_filename, 'ab') as outfile:
```
Comment (dongju-chae) [inline test/input_gen/genInput.py:22]: `input_filename` vs. `outfile`? I think the terminology may cause some confusion.

File: test/input_gen/genInput.py (hunk 1 above)
Comment (jijoongmoon) [reply test/input_gen/genInput.py]: fixed. Thanks.

File: test/input_gen/genInput.py (hunk 2 above)
Comment (jijoongmoon) [reply test/input_gen/genInput.py]: fixed. Thanks.

**Pull request merged**